src/codecovio.jl

```diff
@@ -222,11 +222,15 @@ module Codecov
         @debug "Codecov.io API URL:\n" * mask_token(uri_str)
 
         if !dry_run
-            heads   = Dict("Content-Type" => "application/json",
-                           "Accept" => "application/json")
-            data    = to_json(fcs)
-            req     = HTTP.post(uri_str; body = JSON.json(data), headers = heads)
-            @debug "Result of submission:" * String(req)
+            # Tell Codecov we have an upload for them
+            response = HTTP.post(uri_str; headers=Dict("Accept" => "text/plain"))
+            # Get the temporary URL to use for uploading to S3
+            s3url = split(String(response.body), '\n')[2]
```

Comment:
```suggestion
            s3url = get(split(String(response.body), '\n'), 2, "")
            @debug "Result of submission:" * chomp(replace(String(req), s3url => ""))
            startswith(s3url, "https://") || error("Invalid codecov response: $s3url")
```

Reply:
`req` doesn't exist here, I think you meant `response`? Also, won't `String(response.body)` take ownership of the body and leave `String(response)` not informative, or does the `String` method show headers and whatnot?

Reply:
ah, true, it does add headers, but the other call does destroy the body
```suggestion
            repr = String(response)
            s3url = get(split(String(response.body), '\n'), 2, "")
            repr = chomp(replace(repr, s3url => ""))
            @debug "Result of submission:" * repr
            startswith(s3url, "https://") || error("Invalid codecov response: $s3url")
```